### PR TITLE
Write Google Cloud push step integration

### PR DESCRIPTION
**Pull request by flomotlik**

### CODE REVIEW

File: _posts/docker/deployments/2015-08-31-google-cloud.md [hunk 1]
```diff
@@ -96,7 +96,32 @@ Now you have a working integration with the Google Cloud that will automatically
 
 ## Google Container Engine
 
-Our container also works with the Google Container Engine and Container Registry. The following script will first authenticate with Google, then push a container according to the [Google Container Registry documentation](https://cloud.google.com/container-registry/) and then interact with kubectl to start the service.
+Our container also works with the Google Container Engine and Container Registry. At first we will set up a [push step]({{ site.baseurl }}{% post_url docker/tutorials/2015-08-12-docker-pull %}) to push a container to the registry. Then a script will authenticate with Google and interact with kubectl to start the service.
```
Comment (bfosberry) [inline _posts/docker/deployments/2015-08-31-google-cloud.md]: I think this url should be 

```
docker/tutorials/2015-07-03-docker-push
```

File: _posts/docker/deployments/2015-08-31-google-cloud.md [hunk 2]
```diff
@@ -96,7 +96,32 @@ Now you have a working integration with the Google Cloud that will automatically
...
+
+
+At first we define the gcr_dockercfg service container in the ***codeship-services.yml*** file which will create temporary credentials for us. You can check out the code behind the container in the [codeship-library/gcr-dockercfg-generator](https://github.com/codeship-library/gcr-dockercfg-generator) repository. Make sure the environment variables mentioned above are set up in the encrypted_env_file.
```
Comment (bfosberry) [inline _posts/docker/deployments/2015-08-31-google-cloud.md]: I would change this a little to reference the generic dockercfg generator services we support, something like this:

```
At first we define the [dockercfg generator service]({{ site.baseurl }}{% post_url dockertutorials/2015-12-17-dockercfg-services %}) container in the ***codeship-services.yml*** file which will create temporary credentials for us.
```

**Comment by bfosberry** [comment]
A few comments, but LGTM pending changes, thanks!

File: _posts/docker/deployments/2015-08-31-google-cloud.md [hunk 3]
```diff
@@ -96,7 +96,32 @@ Now you have a working integration with the Google Cloud that will automatically
 
 ## Google Container Engine
 
-Our container also works with the Google Container Engine and Container Registry. The following script will first authenticate with Google, then push a container according to the [Google Container Registry documentation](https://cloud.google.com/container-registry/) and then interact with kubectl to start the service.
+Our container also works with the Google Container Engine and Container Registry. At first we will set up a [push step]({{ site.baseurl }}{% post_url docker/tutorials/2015-08-12-docker-pull %}) to push a container to the registry. Then a script will authenticate with Google and interact with kubectl to start the service.
+
+
```
Comment (mlocher) [inline _posts/docker/deployments/2015-08-31-google-cloud.md]: Please remove the additional new line.

File: _posts/docker/deployments/2015-08-31-google-cloud.md (hunk 1 above)
Comment (flomotlik) [inline _posts/docker/deployments/2015-08-31-google-cloud.md]: yup, thats correct

File: _posts/docker/deployments/2015-08-31-google-cloud.md [hunk 4]
```diff
@@ -96,7 +96,32 @@ Now you have a working integration with the Google Cloud that will automatically
...
+At first we define the gcr_dockercfg service container in the ***codeship-services.yml*** file which will create temporary credentials for us. You can check out the code behind the container in the [codeship-library/gcr-dockercfg-generator](https://github.com/codeship-library/gcr-dockercfg-generator) repository. Make sure the environment variables mentioned above are set up in the encrypted_env_file.
+
+```yaml
+gcr_dockercfg:
+  image: codeship/gcr-dockercfg-generator
+  add_docker: true
+  encrypted_env_file: environment.encrypted
+```
+
+Now we can reference the service in the push step configuration and set up the deployment to the Google registry.
```
Comment (mlocher) [inline _posts/docker/deployments/2015-08-31-google-cloud.md]: can we switch _Google registry_ to _Google Container Registry_?

**Comment by mlocher** [comment]
One more minor comment, :+1: with or without the changes.

**Pull request merged**